packages/orbit-components/src/Skeleton/Skeleton.stories.tsx

```diff
@@ -18,129 +15,113 @@ enum PRESETS {
   Text = "Text",
 }
 
-export const Default = () => {
-  return <Skeleton height="150px" width="500px" />;
+const meta: Meta<typeof Skeleton> = {
+  title: "Skeleton",
+  component: Skeleton,
 };
 
-Default.story = {
+export default meta;
+type Story = StoryObj<typeof Skeleton>;
+
+export const Default: Story = {
+  render: () => <Skeleton height="150px" width="500px" />,
+
   parameters: {
     info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      disable: true,
+    },
   },
 };
 
-export const Playground = ({
-  animate,
-  height,
-  maxHeight,
-  rowBorderRadius,
-  rowHeight,
-  rowOffset,
-  rows,
-  spaceAfter,
-  viewBox,
-  width,
-  title,
-  color,
-}) => {
-  return (
-    <Skeleton
-      animate={animate}
-      color={color}
-      height={height}
-      maxHeight={maxHeight}
-      title={title}
-      rowBorderRadius={rowBorderRadius}
-      rowHeight={rowHeight}
-      rowOffset={rowOffset}
-      rows={rows}
-      spaceAfter={spaceAfter}
-      viewBox={viewBox}
-      width={width}
-    />
-  );
-};
-
-Playground.story = {
+export const Playground: Story = {
```

Comment:
`Preset` arg is missing in Playground story.

Reply:
I didn't include it there, as it would rewrote the other args so it would be as in the other stories when setting `mediumMobile`, `tablet` etc. that it overwrote the individual properties. There is a separate story for `Preset`.